zerver/tests/fixtures/ldap/directory.json

```diff
@@ -77,5 +77,16 @@
         "uid": ["user2_with_shared_email"],
         "sn": ["shortname"],
         "mail": ["shared_email@zulip.com"]
+    },
+
+    "ou=groups,dc=zulip,dc=com": {
+        "ou": "groups"
+    },
+    "cn=cool_test_group,ou=groups,dc=zulip,dc=com": {
+        "objectClass": ["groupOfUniqueNames"],
+        "cn": ["cool_test_group"],
+        "uniqueMember": [
+            "uid=hamlet,ou=users,dc=zulip,dc=com"
+        ]
```

Comment:
Can we add a second test group with multiple members to this, just to show what the structure is?

Reply:
Do you want this second test group to be covered with a unit test as well?

Reply:
I don't think it's essential, in that one can always run it manually, but it could be nice.